kss/builder/decanter/scss/_date-stacked.scss

```diff
@@ -0,0 +1,15 @@
+@charset "UTF-8";
+
+#kssref-components-date-stacked {
+  .su-date-stacked {
+    max-width: 87px;
+
+    @include grid-media-only('md') {
+      max-width: 101px;
+    }
+
+    @include grid-media-only('2xl') {
+      max-width: 105px;
+    }
+  }
+}
```

Comment:
Should these max-widths be limited to the styleguide's rendering? Don't we want a max-width on the actual rendering?

Reply:
I thought a little about this one, and wasn't sure if the max-width should be tied to the component itself or just the style guide for display purposes only. I wondered if the components use case might dictate what width someone wants to set for it. That said, we might actually want a sensible default established; one that they can then override. I'll move this.

Reply:
I do think the widths are inherent in the component. But maybe we could do this styling with a placeholder so people can override if necessary?

Reply:
That is a really good question. My initial knee jerk reaction is to leave components at 100% width with the idea that they will be used either in some sort of container or by a developer in which they could then set the widths. What are other style guides doing? Are they using atomic design principles? Are they applying widths to every level (atom -> organism)? I am sure somebody has gone through some learning lessons on this already and it would be good of us to find out.

Reply:
A couple of examples to digest: 
* http://walmartlabs.github.io/web-style-guide/#responsive-tile
* http://walmartlabs.github.io/web-style-guide/#grid-tile
* https://rizzo.lonelyplanet.com/styleguide/ui-components/cards#styleguide-block__item--1
* https://patterns.wvu.edu/patterns
*